ably/export_test.go

```diff
@@ -1,5 +1,13 @@
 package ably
 
-var BuildPath = func(p *PaginatedResource, base, rel string) (string, error) {
+func (p *PaginatedResource) BuildPath(base, rel string) (string, error) {
 	return p.buildPath(base, rel)
 }
+
+func (opts *ClientOptions) RestURL() string {
+	return opts.restURL()
+}
+
+func (opts *ClientOptions) RealtimeURL() string {
+	return opts.RealtimeURL()
+}
```

Comment:
This is defined only for tests - if tests are external (the ones with _test suffix) and for some reason need to access unexported symbols the pattern is to define them exported for tests, usually in `export_test.go` file. As I removed a number of exported functions from the API I needed to export them for tests. I aim to not use unexported functions in rewritten tests when possible.